js/tic_tac_toe.js

```diff
@@ -1,2 +1,140 @@
-// Your code here
+
 // Use the jQuery
+var App = (function() {
+
+  var $boxCell, $box1, $box2, $box3, $box4, $box5, $box6, $box7, $box8, $box9;
```

Comment:
I think storing these boxes in an array would make your life easier than keeping them all separate.